trade-finance-manager-api/src/v1/controllers/utilisation-reports/fee-record-correction/post-fee-record-correction.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { isAxiosError, HttpStatusCode } from 'axios';
+import { Response } from 'express';
+
+import api from '../../../api';
+import { CustomExpressRequest } from '../../../../types/custom-express-request';
+import { TfmSessionUser } from '../../../../types/tfm-session-user';
+
+export type PostFeeRecordCorrectionRequestBody = {
+  user: TfmSessionUser;
+};
+
+export type PostFeeRecordCorrectionRequest = CustomExpressRequest<{
+  params: {
+    reportId: string;
+    feeRecordId: string;
+  };
+  reqBody: PostFeeRecordCorrectionRequestBody;
+}>;
+
+/**
+ * Creates a new fee record correction
+ * @param req - The request object
+ * @param res - The response object
+ */
+export const postFeeRecordCorrection = async (req: PostFeeRecordCorrectionRequest, res: Response) => {
+  try {
+    const { reportId, feeRecordId } = req.params;
+    const { user } = req.body;
```

Comment:
We have really good validation in central-api and error handling here, so I don't think there is any value in doing so since we don't do anything with these here other than pass them on the request to central-api so if they don't exist the code won't fall over, we will just pass through `undefined` and then the validation in central will pass up the appropriate error for any missing required parameters.